SpatialGDK/Documentation/content/get-started/dependencies.md

```diff
@@ -42,6 +42,9 @@ To build the GDK for Unreal you need the following software installed on your ma
       - You must also select the **.NET Framework 4.6.2 development tools**
     - **Desktop development with C++**<br>
     - **Game development with C++**, including the optional **Unreal Engine installer** component.
+
+<%(Callout type="warn" message="Please make sure you install Visual Studio **2017**. The GDK does not currently support any other version.")%>
```

Comment:
```suggestion
<%(Callout type="warn" message="Make sure you install Visual Studio 2017. The GDK does not currently support any other version.")%>
```

(Making two suggestions that I think Laura would otherwise make if she'd got here first!)